Public/Src/Cache/ContentStore/Distributed/NuCache/EventStreaming/EventHubContentLocationEventStore.cs

```diff
@@ -17,6 +17,7 @@
 using Microsoft.Azure.EventHubs;
 using Microsoft.Practices.TransientFaultHandling;
 using static BuildXL.Cache.ContentStore.Distributed.NuCache.EventStreaming.ContentLocationEventStoreCounters;
+using Exception = System.Exception;

```

Comment:
Someone named a type Exception?! Or is there some other reason this is required?

Reply:
Removed. It seems that R# inserted this, but for no good reason.